LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -258,4 +249,50 @@ class LockboxXCUITests: BaseTestCase {
         XCTAssertTrue(app.cells["appVersionSettingOption"].exists)
         XCTAssertNotEqual(app.cells.staticTexts.element(boundBy: 2).label, "")
     }
+
+    // Verify SetAutofillNow
+    func testSetAutofillNow() {
+        // Disconnect account
+        disconnectAndConnectAccount()
+        if #available(iOS 12.0, *) {
+            waitforExistence(app.buttons["setupAutofill.button"])
+            app.buttons["setupAutofill.button"].tap()
+        }
+        let settingsApp = XCUIApplication(bundleIdentifier: "com.apple.Preferences")
+        waitforExistence(settingsApp.cells.staticTexts["Passwords & Accounts"])
+    }
+
+    // Once app is open
+    func testSetAutofillSettings() {
+        // Open Setting to add Lockbox
+        navigator.goto(Screen.SettingsMenu)
+        navigator.performAction(Action.OpenDeviceSettings)
+        // Wait until settings app is open
+        let settingsApp = XCUIApplication(bundleIdentifier: "com.apple.Preferences")
+        waitforExistence(settingsApp.cells.staticTexts["Passwords & Accounts"])
+        // Configure Passwords & Accounts settings
+        settingsApp.cells.staticTexts["Passwords & Accounts"].tap()
+        settingsApp.cells.staticTexts["AutoFill Passwords"].tap()
+        settingsApp.switches["AutoFill Passwords"].tap()
+        waitforExistence(settingsApp.cells.staticTexts["Lockbox"])
+        settingsApp.cells.staticTexts["Lockbox"].tap()
+        // Wait until the app is updated
+        sleep(5)
+        settingsApp.terminate()
+
+        // Open Safari
+        let safari = XCUIApplication(bundleIdentifier: "com.apple.mobilesafari")
+        safari.launch()
+        waitforExistence(safari.buttons["URL"], timeout: 5)
+        waitforExistence(safari.textFields["Email or phone"])
+        safari.textFields["Email or phone"].tap()
+        // Need to confirm what is shown here, different elements have appeared and
+        if (safari.otherElements["Password Auto-fill"].exists) {
+            safari.otherElements["Password Auto-fill"].tap()
+        }
+        safari.buttons["Use “iosmztest@gmail.com”"].tap()
+        // Once previous is clear we can assert that the element shown is correct
+        //XCTAssertTrue(safari.buttons["iosmztest@gmail.com, for this website — Lockbox"].exists)
```

Comment:
Why is this commented out?

Reply:
During testing I has seeing to different screen when autofill was suggested:

![lockboxautofill1](https://user-images.githubusercontent.com/1897507/46004632-a4b7fb80-c0b3-11e8-85fd-4d5ce0c47517.png)
![lockboxautofill2](https://user-images.githubusercontent.com/1897507/46004640-a97caf80-c0b3-11e8-84be-8f8e424c2f3d.png)


So I commented the assert until it was clear which one was shown and it was always being shown so there were not false failures.
I will check the app and try to uncomment that line if that is clear now.